.github/release-drafter.yml

```diff
@@ -1,4 +1,4 @@
-name-template: 'v$RESOLVED_VERSION 🧪'
```

Comment:
not a very important thing, but could we maintain our own 'release-drafter-yml' for the self pipeline? would be a good sample to copy from for those that our looking for an own template.

Reply:
Hi @ivy-rew , I recently add another template for self pipeline - just make it same as previous version of release-drafter.yaml here. Please help me to check it.